### PR TITLE
 Innovation Disclosure Assistant

### PR DESCRIPTION
**Pull request by davethenipper**
Adding a page about the Innovation Disclosure Assistant.

### CODE REVIEW

**Comment by Mctalian** [comment]
Approved

**Comment by YashdalfTheGray** [comment]
File names follow camelCase, so rename `InnovationDisclosure` to `innovationDisclosureView` for both the folder and the tpl.html.

File: src/app.js [hunk 1]
```diff
@@ -52,6 +52,11 @@ angular.module('flintAndSteel', [
                 url: '/about',
                 templateUrl: 'about/about.tpl.html',
                 controller: ''
+            })
+            .state('InnovationDisclosure', {
```
Comment (YashdalfTheGray) [inline src/app.js]: States have been named in all lower case so far, can you change it to `innovationdisclosure`?

**Comment by davethenipper** [comment]
Ty sir.  Should be good to go now.

**Comment by YashdalfTheGray** [comment]
Approved.

**Comment by alanlgirard** [comment]
Approved.

**Comment by davethenipper** [comment]
Merged!

**Pull request merged**